uniffi_bindgen/src/scaffolding/mod.rs

```diff
@@ -48,7 +48,7 @@ mod filters {
             Type::Optional(t) => format!("std::option::Option<{}>", type_rs(t)?),
             Type::Sequence(t) => format!("std::vec::Vec<{}>", type_rs(t)?),
             Type::Map(k, v) => format!(
-                "std::collections::HashMap<{}, {}>",
```

Comment:
UHHHH this snuck in during the merge conflict I had when I was merging main into the `release-0.19.x` branch. I'll cut 0.19.5.